apps/omg_eth/lib/omg_eth/supervisor.ex

```diff
@@ -26,12 +26,17 @@ defmodule OMG.Eth.Supervisor do
   end
 
   def init(:ok) do
+    check_interval_ms = Application.fetch_env!(:omg_eth, :ethereum_events_check_interval_ms)
```

Comment:
adding this to `OMG.Eth.Configuration`? wdyt?

Reply:
Sounds good. A new PR dedicated to converting configurations!